src/CallWebhookJob.php

```diff
@@ -70,13 +73,18 @@ public function handle()
                 'timeout' => $this->requestTimeout,
                 'verify' => $this->verifySsl,
                 'headers' => $this->headers,
-            ], $body));
+                'on_stats' => function (TransferStats $stats) {
+                    
+                    $this->response = $stats->getResponse();
```

Comment:
Do we have to do this, the response is already captured on line 72, no?

Reply:
It is redundant I've removed this and just set the entire `TransferStats` instead.